lib/charms/hpc_libs/v0/slurm_ops.py

```diff
@@ -461,7 +461,15 @@ def install(self) -> None:
         repositories.add(experimental)
 
         apt.update()
-        for package in ["slurm-wlm", "mungectl", "prometheus-slurm-exporter"]:
+        for package in [
+            "slurmctld",
+            "slurmd",
+            "slurmdbd",
+            "slurmrestd",
+            "slurm-client",
+            "mungectl",
+            "prometheus-slurm-exporter",
+        ]:
```

Comment:
After merging the `AptManager` I realized that `apt.add_package` already puts on its error the list of packages that fail to get installed, so I think we can move this array to be inside `apt.add_package`, maybe using a generic message for the raised `SlurmOpsError` instead of printing the package.